tutorials/appengine-yarn/index.md

```diff
@@ -1,34 +1,39 @@
 ---
-title: Using yarn on App Engine
-description: Learn how to use yarn to manage dependencies for node.js applications on Google App Engine
+title: Using Yarn on App Engine
+description: Learn how to use Yarn to manage dependencies for Node.js applications on Google App Engine flexible environment.
```

Comment:
when/if that day comes, we will update all of the App Engine tutorials in this repo. Until then, we decided that each App Engine tutorial would call out whether it pertained to standard or flexible environment.